beginner_source/blitz/autograd_tutorial.py

```diff
@@ -3,6 +3,14 @@
 Autograd: Automatic Differentiation
 ===================================
 
+**Video By**: `Elliot Waite`_
+
+.. raw:: html
+
+   <div style="margin-top:10px; margin-bottom:10px;">
+     <iframe width="560" height="315" src="https://www.youtube.com/embed/MswxJw-8PvE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Maybe remove the "autoplay; " part. That was the only difference I noticed between this code and this pull request where a video was added to the top of the 60 Min Blitz Tutorial: https://github.com/pytorch/tutorials/commit/fb00d7827a7d7646390f6dc484684f303bd76789

Reply:
Thanks. You crushed the bug, without any error logs. All tests passed, but still I cannot see the changes on [this](https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html) page. I think the changes will take effect in the next release.

Reply:
Okay. I see the changes need to be approved. This is unmerged pull request until then.